src/database/user_operations.py

```
@@ -1,144 +1,134 @@
-                (username, ah, mu, bid)
-            )
-            
-            # Права доступа
-            await cur.execute(
-                "INSERT INTO account_access(id,gmlevel,RealmID) VALUES(%s,3,-1)", (bid,)
-            )
-            
-            # Связка с Telegram
+
+            # Account hash (SHA1 username:password)
+            account_hash = hashlib.sha1(f"{username}:{pu}".encode()).hexdigest().upper()
```

Ruff (0.12.2)

Probable use of insecure hash functions in `hashlib`: `sha1`

(S324)